src/shared/views/plans/planItinerary/PlanMap/PlanMap.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import styles from './PlanMap.css'
+
+const PlanMap = () => {
+  return (
+    <div>
+      <h2 className={styles.title}>RECOMMENDED ROUTE</h2>
+      <img className={styles.map} alt="map" src="/static/maps/Level2-plan.jpg" />
```

Comment:
@nickmask Can we upload (just straight to master) a version using a different color for the path. Yellow on yellow doesnt work so well.